repos.d/kiss/noir.yaml

```diff
@@ -0,0 +1,33 @@
+###########################################################################
+# NOIR Linux
+###########################################################################
+- name: noir
+  type: repository
+  desc: NOIR Linux Main
+  family: noir
```

Comment:
You also need to list this in [repology-schemacheck.py](repology-schemacheck.py)

Reply:
done.